packages/yoroi-extension/app/api/ada/index.js

```diff
@@ -2544,7 +2544,11 @@ function getDifferenceAfterTx(
   stakingKey: RustModule.WalletV4.PublicKey,
   defaultToken: DefaultTokenEntry,
 ): MultiToken {
-  const stakeCredential = RustModule.WalletV4.StakeCredential.from_keyhash(stakingKey.hash());
+
+  const accountKeyString = RustModule.WasmScope(Scope => {
+    const stakeCredential = Scope.WalletV4.StakeCredential.from_keyhash(stakingKey.hash());
+    return Buffer.from(stakeCredential.to_bytes()).toString('hex')
```

Comment:
@AhmedIbrahim336 , we can use CSL built-in `.to_hex()` and `.from_hex(str)` functions now.

```suggestion
    return stakeCredential.to_hex();
```